resources/css/core.scss

```diff
@@ -6,12 +6,6 @@ $roboto-font-path: "~roboto-fontface/fonts" !default;
 @import "./common/styles";
 @import "~bootstrap/scss/bootstrap";
 
-@each $color, $value in $theme-colors {
```

Comment:
Be careful with removing this since apps might depend on these colors. Probably not, but this could have an effect.

Reply:
I'll revert this. The launcher style file is no longer importing core.scss.